controllers/flowlogspipeline/metrics_definitions/workload_egress_bytes_total.yaml

```diff
@@ -6,16 +6,15 @@ details:
 usage:
   Evaluate network outgoing usage breakdown per source and destination namespaces and owners
 tags:
-  - bandwidth
-  - rate
-  - namespace
-  - owner
-  - outgoing
+  - egress
+  - bytes
+  - workloads
+  - namespaces
```

Comment:
I wouldn't set "namespaces" here, else if we later add namespace-only metrics later, we wouldn't be able to filter them out without impacting this one as well (or we'd need to find a trick to do it)

Reply:
well, depends how we do the namespace-only metrics ... you plan to add label-removal feature like we discussed previously, right? If so, then it's ok for me

Reply:
I plan to do the label-removal, but I am not sure that this is the priority.

I removed the namespaces label and I will add it again once the label removal task is done.